lib/mod.lua

```diff
@@ -9,12 +10,25 @@ local my_midi = {
 function my_midi:send(data) end
 function my_midi:note_on(note, vel, ch)
     if ch == 1 then
+        if not channel_is_set_up then
+            print("Refusing to access nbout_chan_1")
```

Comment:
Can we change the message to "nbout attempted to send a note without initializing" or something?

Reply:
Well, the issue is that the sequencer is sending a note before nbout has initialized, so how about "nbout received note_on before initialization"?